packages/components/src/dropdown-menu/style.scss

```diff
@@ -30,19 +30,30 @@
 			height: 1px;
 		}
 
-		&.is-active svg {
-			// Block UI appearance.
-			color: $white;
-			background: $gray-900;
-			box-shadow: 0 0 0 $border-width $gray-900;
-			border-radius: $border-width;
+		&.is-active {
+			svg,
+			span.dashicon {
+				// Block UI appearance.
+				color: $white;
+				background: $gray-900;
+				box-shadow: 0 0 0 $border-width $gray-900;
+				border-radius: $border-width;
+			}
 		}
 
 		// Formatting buttons
-		> svg {
+		> svg,
+		> span.dashicon {
```

Comment:
Increased specificity as `span.dashicon` instead of `dashicon` to override some of [the default styles for the button component](https://github.com/WordPress/gutenberg/blob/e5d9a436d5426aeb19e9de910d52505b55edce36/packages/components/src/button/style.scss#L302-L326).